### PR TITLE
dtr backup warning added

### PR DESCRIPTION
**Pull request by stevenhanna6**
dtr backup warning added

Fixes: #2986

### CODE REVIEW

**Comment by stevenhanna6** [comment]
I don't know if that's enough, just a warning before the 1st step?

**Comment by mstanleyjones** [comment]
@joaofernandez ptal

File: datacenter/dtr/2.2/guides/admin/upgrade.md [hunk 1]
```diff
@@ -45,6 +45,12 @@ Before starting your upgrade planning, make sure that the version of UCP you are
 using is supported by the version of DTR you are trying to upgrade to. <!--(TODO:
 link to the compatibility matrix)-->
 
+> **Warning**
+>
+> Before performing any upgrade it’s important to backup. See 
+> [DTR backups and recovery](/datacenter/dtr/2.2/guides/admin/backups-and-disaster-recovery/.md).
```
Comment (joaofnfernandes) [inline datacenter/dtr/2.2/guides/admin/upgrade.md]: Can you link to `/datacenter/dtr/2.2/guides/admin/backups-and-disaster-recovery/`
 instead? It provides a bit more guidance than just the reference.

File: datacenter/dtr/2.1/guides/install/upgrade.md [hunk 2]
```diff
@@ -13,6 +13,12 @@ There is no downtime when upgrading an highly-available DTR cluster. If your
 DTR deployment has a single replica, schedule the upgrade to take place outside
 business peak hours to ensure the impact on your business is close to none.
 
+> **Warning**
```
Comment (joaofnfernandes) [inline datacenter/dtr/2.1/guides/install/upgrade.md]: Can you link to `/datacenter/dtr/2.1/guides/high-availability/backups-and-disaster-recovery/`. It provides more guidance than the reference?

**Comment by stevenhanna6** [comment]
changes made, hmmm I don't know if I had to rebase I did the fixes to fast before pushing the fixes. Let me know if I screwed it up, still new with this git stuff

**Pull request merged**